shared/js/browser/macos-communication.js

```diff
@@ -73,7 +73,7 @@ const combineSources = () => ({
     tab: Object.assign(
         {},
         trackerBlockingData || {},
-        { maliciousSiteStatus: maliciousSiteStatus ?? false },
+        { maliciousSiteStatus: maliciousSiteStatus ?? null },
```

Comment:
I had some bugs on android when using `??=` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment - (basically there are some old  webviews out there in the wild). I’d either check the compiled output to see if esbuild is down-levelling this syntax, or change it